docs/blog/2019-03-27-security-for-modern-web-frameworks/index.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Security for Modern Web Frameworks.
+date: 2019-03-27
```

Comment:
@DSchau I merged this post without checking the date, do we need to do anything else for this one to be published? (does it need to go into scheduling?)

Reply:
It's already live on the site...it just shows as being backdated.

Reply:
Well hey, that's fine by me if you're okay with it. :) congrats on publishing your post!